### PR TITLE
[FRENCH] Territorial scope of the GDPR

### PR DESCRIPTION
**Pull request by eliottha**
Translated Territorial scope of the GDPR to French.

### CODE REVIEW

**Comment by baltpeter** [comment]
Hey @eliottha,

thanks for bringing this up. For the discussion on why we went with the informal form, see https://github.com/datenanfragen/website/issues/148#issuecomment-544135208. We also use the informal form for the German translation but as I only know some basic French from school, I left that decision to @NathanBnm.

In the German translation, we explicitly go for language similar to what you would use between friends. But again, I don't know French well enough to judge that there. Considering we already went with demande**tes**donnees.fr for the name, I think switching to vous now would sound weird, though. What do you think?

**Comment by eliottha** [comment]
@baltpeter can u send the adapted image.

File: content/fr/blog/gdpr-territorial-scope/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Portée Territoriale du RGPD",
+    "date": "2020-10-31T21:22:15+02:00",
+    "type": "blog",
+    "description": "Qui est habilité à faire valoir les droits des personnes concernées par le RGPD ? Et contre qui ? Voici un aperçu de la portée territoriale du RGPD.",
+    "featured_image": "gdpr-territorial-scope",
+    "tags": ["rgpd", "portée territoriale", "rgpd article 3" ],
+    "authors": [ "eliottha" ],
```
Comment (baltpeter) [inline content/fr/blog/gdpr-territorial-scope/index.md:8]: Please add yourself to [this file](https://github.com/datenanfragen/website/blob/master/config/_default/params.toml), so we can list you as an author. You need to specify a short and a long name (the long name can optionally include a link to a website/social media profile/your GitHub account/etc. if you want).  
You of course don't need to use your real name, pseudonyms or user names are also absolutely fine.

Please also leave `"anikarini"` in the list as she originally wrote the post:

```suggestion
    "authors": [ "anikarini", "eliottha" ],
```

Comment (baltpeter) [reply content/fr/blog/gdpr-territorial-scope/index.md:8]: By the way, feel free to also add yourself to the [`AUTHORS` file](https://github.com/datenanfragen/website/blob/master/AUTHORS) if you want. That way, you are listed on our [open source page](https://www.datarequests.org/open-source#contributors) as a contributor.

File: content/fr/blog/gdpr-territorial-scope/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Portée Territoriale du RGPD",
+    "date": "2020-10-31T21:22:15+02:00",
+    "type": "blog",
+    "description": "Qui est habilité à faire valoir les droits des personnes concernées par le RGPD ? Et contre qui ? Voici un aperçu de la portée territoriale du RGPD.",
+    "featured_image": "gdpr-territorial-scope",
```
Comment (baltpeter) [inline content/fr/blog/gdpr-territorial-scope/index.md:6]: I'll upload a translated image.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thank you very much!

File: content/fr/blog/gdpr-territorial-scope/index.md (hunk 1 above)
Comment (eliottha) [reply content/fr/blog/gdpr-territorial-scope/index.md]: Added myself, thank you!

**Review by baltpeter — APPROVED** [review]
Great, thanks!

**Pull request merged**